testgrid/config/config.yaml

```diff
@@ -1990,8 +1990,40 @@ dashboards:
   - name: periodic-kubeadm-gce-1.6
     test_group_name: periodic-kubernetes-e2e-kubeadm-gce-1-6
 
+# These are the release *blocking* tests.  These provide a valid binary signal
+# to gate releases (alpha, beta, official).
+# This list is used by kubernetes/release/lib/releaselib.sh to provide no/nogo
+# signal for releases.
+# The first test/suite in the list is the primary suite for release purposes
+- name: release-1.6-blocking-hard
```

Comment:
maybe leave the name of the "blocking-hard" groups as-is, so we don't break bookmarks?